test/Microsoft.AspNetCore.Mvc.IntegrationTests/CancellationTokenModelBinderIntegrationTest.cs

```diff
@@ -50,7 +50,9 @@ private class Person
             // Model
             var boundPerson = Assert.IsType<Person>(modelBindingResult.Model);
             Assert.NotNull(boundPerson);
-            Assert.NotSame(default(CancellationToken), boundPerson.Token);
```

Comment:
This one was a false positive. NotSame is always going to pass for value types. Testing the behavior required actually triggering the request abort to ensure the token was assigned as expected.